pipeline/processor/postprocessor.py

```diff
@@ -720,4 +720,5 @@ def process(self, document: Document, filename: Optional[str]) -> Document:
         doc = document.to_dict()
         new_contents = self._group_list_elements(doc)
         new_contents["filename"] = filename
```

Comment:
Slight hack here to get around a bit of an antipattern in the post-processor that will results in missing md5 hashes.